editor/editor_audio_buses.cpp

```diff
@@ -376,14 +383,23 @@ float EditorAudioBus::_scaled_db_to_normalized_volume(float db) {
 }
 
 void EditorAudioBus::_show_value(float slider_value) {
-	String text = vformat("%10.1f dB", _normalized_volume_to_scaled_db(slider_value));
+
+	float db;
+	if (Input::get_singleton()->is_key_pressed(KEY_CONTROL)) {
+		// Display the correct (snapped) value when holding Ctrl
```

Comment:
I didn't check where this method is called exactly, but does it really make sense to snap the *display* value on a keypress? It makes sense to use Ctrl to snap what will be set, but as for showing the value, it should likely show the real value set for the bus. If it's no longer round due to precision issues on the conversion, that's another issue but I don't think we should try to hide it.

Reply:
I snapped it so it displays the value that will be set once you release the slider. Otherwise, the tooltip could display something like `-2.4 dB` but the real value would be `-2 dB` (which you would see after hovering the slider again).

Reply:
Ah ok, that makes sense.